### PR TITLE
Update installconfig-regiond.md

### PR DESCRIPTION
**Pull request by roaksoax**
Update to add a section on how to increase the number of workers

### CODE REVIEW

File: en/installconfig-region.md [hunk 1]
```diff
@@ -59,3 +59,22 @@ Check the log files for any errors:
...
+
+This is possible by customizing ```regiond.conf (/etc/maas/regiond.conf)``` and setting ```num_workers```. For example:
+
+```
+[...]
+num_workers: 8
+```
+
+!!! Note:
+    Modifying the number of workers will increase the number of database connections. This requires PostgreSQL to have an increased number of allowed connections. Please see [High Availability][PostgreSQL HA] for more information.
```
Comment (blakerouse) [inline en/installconfig-region.md]: Would be good to state that for every worker added the postresql connection limit needs to be increased by 11. As each worker will use at most 11 postgresql connections.

File: en/installconfig-region.md [hunk 2]
```diff
@@ -59,3 +59,22 @@ Check the log files for any errors:
...
+
+The MAAS Region Controller is a daemon collection of 4 workers that are in charge of handling all the internals of MAAS. The regiond workers handle the UI, API and the internal communication between Region and Rack controllers.
+
+Increasing the number of worker in larger environments where there are multiple rack controllers should increase the performance of the Region by allowing more workers to handle internal communication between the Region and the Rack controllers.
+
+This is possible by customizing ```regiond.conf (/etc/maas/regiond.conf)``` and setting ```num_workers```. For example:
+
+```
+[...]
+num_workers: 8
```
Comment (blakerouse) [inline en/installconfig-region.md]: It's best to set this to the same number of CPU's the system has. Each worker is single threaded, so setting it to 1 worker per CPU will provide the best performance.

Comment (roaksoax) [reply en/installconfig-region.md]: do we pin to a CPU? what if we have 80 cores? it doesn't make sense to do that....

Comment (blakerouse) [reply en/installconfig-region.md]: No it doesn't pin to a CPU, and yes 80 cores.... probably doesn't make sense, but 16 would. I wouldn't set it above 16. Everything with in reason of course... common sense.

I can go to an all you can eat buffet, but doesn't mean I am going to eat all I can possible fit in me until I explode. ;-)

**Pull request merged**